### ISSUE TITLE
Possible error with model implementation

### ISSUE THREAD

**Issue by maxzuo** [maustinstar/IRIS#3]
Model taking too long to run on iPhone. Testing other models out there of larger sizes, the model should not take as long as it does to make a prediction.

Possible sources of problem:
* Model created in Keras not done properly (issue with architecture).
* Model implementation within Swift not optimized
* Model size too large.

**Comment by maxzuo** [comment]
Issue with architecture resolved by switching to almost an entirely new architecture.
* SRCNN went too deep too quickly. By adding more convolutional layers but gradually reaching 64 filters, the new model can help decrease the amount of time spent.
* New model's PSNR accuracy should also be improved.
* Further experimentation with PyTorch required. Model speed still not optimal.
